### DISCUSSION TITLE
How to set arguments for debugables?

### DISCUSSION THREAD

**Discussion by alejandrodnm** [mrcjkb/rustaceanvim#117] [Q&A] [answered]
I'm trying to set the arguments for debugging a binary. I managed it by creating a custom launch.json like:

```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "lldb",
      "request": "launch",
      "name": "Debug Rust Project",
      "program": "${workspaceFolder}/target/debug/hypershift",
      "args": [
        "clone",
        "-c",
        "__adn.hypershift.yml"
      ],
      "cwd": "${workspaceFolder}"
    }
  ]
}
```

Is this the prefer way? 

My concern is that this configuration could be different than the default the plugin uses, and that I might be missing something.

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

The default configuration provided by this plugin is very simple; here's the equivalent JSON:

```json
{
  "name": "Rust debug client",
  "type": "lldb",
  "request": "launch",
  "stopOnEntry": false
}
```

So I think your launch.json should be fine.

**Reply by alejandrodnm** [reply]
Awesome :) thanks. 

BTW, and totally unrelated, there's been some work for replacing the rust-tools with rustaceanvim in LazyVim https://github.com/LazyVim/LazyVim/pull/2198
